src/internal_modules/roc_core/heap_arena.cpp

```diff
@@ -16,10 +16,13 @@
 namespace roc {
 namespace core {
 
-int HeapArena::enable_leak_detection_ = false;
+int HeapArena::enable_leak_detection_ =
+    (DefaultHeapArenaFlags & HeapArenaFlag_EnableLeakDetection) != 0;
+int HeapArena::enable_guards_ = (DefaultHeapArenaFlags & HeapArenaFlag_EnableGuards) != 0;
```

Comment:
Can these be bools? Is there any build problems when using AtomicOps on bools?

Reply:
> Can these be bools?

Probably not on all platforms where we use libatomic_ops instead of C11 __atomic funcs.

In this case it's better to use a single atomic that will hold all flags, instead of seperate atomic for each flag.